### PR TITLE
feat(service-portal): New design fixes - 2

### PR DESCRIPTION
**Pull request by disaerna**
# Service Portal - new design fixes 

_New pullrequest -  This PR #6200 was keeping commit history from other branch _

## What

* Error screen - image was too large in mobile
* Icon should be collapsable/expandable in sidebar
* Animation on icons in sidebar added

## Why

UX ? 

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/service-portal/src/components/Sidebar/ModuleNavigation.tsx [hunk 1]
```diff
@@ -54,6 +53,10 @@ const ModuleNavigation: FC<Props> = ({
     if (external) servicePortalOutboundLink()
   }
 
+  useEffect(() => {
+    setExpand(isModuleActive)
+  }, [isModuleActive])
```
Comment (darrikonn) [inline apps/service-portal/src/components/Sidebar/ModuleNavigation.tsx:58]: setExpand should also be a dependency here.

File: apps/service-portal/src/components/Sidebar/NavItem/NavItem.css.ts [hunk 2]
```diff
@@ -94,9 +90,22 @@ export const text = style({
...
+  color: theme.color.blue600,
+  ':hover': {
+    color: theme.color.blue400,
+  },
+})
+
 export const icon = style({
   pointerEvents: 'none',
   height: '26px',
+  animation: `250ms ease-in-out 0s 1 ${iconEaseIn}`,
```
Comment (darrikonn) [inline apps/service-portal/src/components/Sidebar/NavItem/NavItem.css.ts:108]: willChange opacity here

**Review by darrikonn — CHANGES_REQUESTED** [review]

File: apps/service-portal/src/components/Sidebar/ModuleNavigation.tsx (hunk 1 above)
Comment (disaerna) [reply apps/service-portal/src/components/Sidebar/ModuleNavigation.tsx]: fixed

File: apps/service-portal/src/components/Sidebar/NavItem/NavItem.css.ts (hunk 2 above)
Comment (disaerna) [reply apps/service-portal/src/components/Sidebar/NavItem/NavItem.css.ts:108]: fixed - I promise to remember "will-change" next time 🥲

File: apps/service-portal/src/components/Sidebar/ModuleNavigation.tsx [hunk 3]
```diff
@@ -110,7 +116,7 @@ const ModuleNavigation: FC<Props> = ({
       {!collapsed && navArray && nav.enabled !== false && (
         <AnimateHeight
           duration={300}
-          height={isModuleActive || alwaysExpanded ? 'auto' : 0}
+          height={expand ? 'auto' : alwaysExpanded ? 'auto' : 0}
```
Comment (thordurhhh) [inline apps/service-portal/src/components/Sidebar/ModuleNavigation.tsx:119]: Everything related to `alwaysExpanded` can be removed now with the new design.
It was only meant to be a temporary solution 🚀

**Review by thordurhhh — CHANGES_REQUESTED** [review]
Just one minor thing. Otherwise looks good.

**Review by darrikonn — APPROVED** [review]

**Review by thordurhhh — APPROVED** [review]
LGTM 👍

**Pull request merged**